docs/js/react.js

```diff
@@ -10,7 +10,7 @@
  * LICENSE file in the root directory of this source tree. An additional grant
  * of patent rights can be found in the PATENTS file in the same directory.
  *
- * 
+ *
```

Comment:
Don't pay attention to those, my editor is configured to trim the whitespaces at the end of the line, I can revert those changes if necessary